### DISCUSSION TITLE
Graph for current branch

### DISCUSSION THREAD

**Discussion by perettigiuliano** [mlange-42/git-graph#82] [Ideas]
Is it possible to have graph only for the current branch?
Sometimes I don't need to see all other commits from other branches (neither main/master)

Thanks

**Comment by peso** [comment]
No, it is not currently possible. Some of the changes that are in the pipeline for the library could be used to implement this feature, but are not currently planned.

Please describe your use case as a feature request in the issues section.
New ideas are always welcome.
